coilsnake/modules/eb/StaffModule.py

```diff
@@ -0,0 +1,268 @@
+import logging
+
+from coilsnake.exceptions.common.exceptions import CoilSnakeError, InvalidUserDataError
+from coilsnake.modules.eb.EbModule import EbModule
+from coilsnake.util.common.assets import open_asset
+from coilsnake.util.common.yml import yml_load
+from coilsnake.util.eb.pointer import from_snes_address, to_snes_address
+
+log = logging.getLogger(__name__)
+
+STAFF_TEXT_POINTER_OFFSET = 0x4f253
+LENGTH_OFFSETS = [0x4f583, 0x4f58c, 0x4f66f]
+
+MODE_CONTROL = 0
+MODE_SMALL   = 1
+MODE_BIG     = 2
+MODE_SPACE   = 3
+
+HEIGHT_SMALL = 8
+HEIGHT_BIG   = 16
+
+CONTROL_END_OF_LINE  = 0x00
+CONTROL_SMALL        = 0x01
+CONTROL_BIG          = 0x02
+CONTROL_SPACE        = 0x03
+CONTROL_PLAYER_NAME  = 0x04
+CONTROL_END_OF_STAFF = 0xff
+
+MAX_BYTE = 0xff - 0x40
+SCREEN_WIDTH_IN_TILES = 32
+
+ENTRY_TYPE = 'Type'
+ENTRY_TILE = 'Top 8x8 tile'
```

Comment:
what does "Top 8x8 tile" mean? is there a shorter/more clear term that could be used here?

Reply:
I have a comment that tries to explain this at the top of eb_staff_chars.yml - "in the 'Top 8x8 tile' field the first hex digit is the row, the second is the column in Fonts/credits.png.". For 8x8 characters it's the whole thing; for 8x16 characters it's the top 8x8. Any suggestions for a different name?